### ISSUE TITLE
Related to base class performances

### ISSUE THREAD

**Issue by JAYATEJAK** [xyutao/fscil#26]
I have taken 100 CUB classes as base classes and learned the restnet-18 network. I followed the same setting (50 epochs) mentioned in paper and It is giving 69% as base task accuracy. But when I started with the same base classes using the NCM method (UCIR- where cosine normalization is applied on class weights and features are l2 normalized) the base task performance is giving around 74%. In the paper, base task performance is mentioned as 68.8% in both methods. Can you explain how it is possible? or any different training setting is needed for the NCM method for base class training?

Thank you.

**Comment by Doflamingo-swj** [comment]
Hello, I'm also interested in this work and would like to do some entry-level research. But since the files are missing and cannot be started, please ask if you have these missing files:
tools.ng_anchor ; 
tools.loss ; 
tools.plot ;  
Please provide me with some ideas and have a good life!
